testsuite/policy/__init__.py

```diff
@@ -20,11 +20,20 @@ class Policy(OpenShiftObject):
     """Base class with common functionality for all policies"""
 
     def wait_for_ready(self):
-        """Wait for a Policy to be Enforced"""
-        success = self.wait_until(has_condition("Enforced", "True"))
+        """Wait for a Policy to be fully Enforced"""
+        success = self.wait_until(
```

Comment:
How about adding `wait_for_full_enforced` method and having `wait_for_ready` just call that method, it might make it clearer when using both `wait_for_full_enforced` and `wait_for_partial_enforced` in one tests

Reply:
yes, that's a good idea, will refactor. Key thing to discuss here is whether we want to use `wait_for_ready` for full enforcement only. It seems that we are in agreement that that's exactly what we want.

Reply:
I think it is a discussion for other PR, but my understanding is that `wait_for_ready` means that is is ready for general testsuite use, which for policies almost always is some kind of enforce, but might differ object to object.